interfaces/openfga/v0/README.md

```diff
@@ -0,0 +1,67 @@
+# `openfga`
+
+## Usage
+
+This relation interface describes the expected behaviour of any charm claiming to be able to interact with a OpenFGA.
+
+In most cases, this will be accomplished using the [openfga library](https://github.com/canonical/cs-openfga/blob/main/charms/openfga-k8s/lib/charms/openfga_k8s/v0/openfga.py), although charm developers are free to provide alternative libraries as long as they fulfil the behavioural and schematic requirements described in this document.
+
+## Direction
+
+```mermaid
+flowchart TD
+    Requirer -- openfga, store-name --> Provider
+    Provider -- openfga, store-id, token, address, scheme, port --> Requirer
+```
+
+As with all Juju relations, the `openfga` interface consists of two parties: a Provider (openfga charm), and a Requirer (application charm). The Requirer will be expected to provide an authentication store name, and the Provider will provide new unique credentials (along with other optional fields), which can be used to access the OpenFGA store.
```

Comment:
```suggestion
As with all Juju relations, the `openfga` interface consists of two parties: a Provider (openfga charm), and a Requirer (application charm). The Requirer will be expected to expose an authentication store name, and the Provider will create and forward new unique credentials (along with other optional fields), which can be used to access the OpenFGA store.
```

tip: avoid using `provide` and `require` in these documents, they're kind of confusing in the context :)